README.md

```diff
@@ -168,6 +168,59 @@ public class Handler implements RequestHandler<APIGatewayV2ProxyRequestEvent, AP
 }
 ```
 
+### Trace/Log Correlations
+
+In order to correlate your traces with your logs, you must inject the trace context
+into your log messages. We've added the these into the slf4j MDC under the key `dd.trace_context`
+and provided convenience methods to get it automatically. The trace context is added to the MDC as a side
+effect of instantiating any `new DDLambda(...)`.
+
+This is an example trace context: `[dd.trace_id=3371139772690049666 dd.span_id=13006875827893840236]`
+
+#### JSON Logs
+
+If you are using JSON logs, add the trace ID and span ID to each log message with the keys 
+`dd.trace_id` and `dd.span_id` respectively. To get a map containing trace and span IDs,
+ call `DDLambda.getTraceContext()`. Union this map with the JSON data being logged.
+
+#### Plain text logs
+
+If you are using plain text logs, then you must create a new [Parser](https://docs.datadoghq.com/logs/processing/parsing/?tab=matcher)
+that can extract the trace context from the correct position in the logs. Use the helper `_trace_context` to
```

Comment:
It's worth noting that this helper is only available if you are modifying our Lambda pipeline, with our existing grok rules.